src/System.IO.Pipelines.Networking.Sockets/System.IO.Pipelines.Networking.Sockets.csproj

```diff
@@ -7,8 +7,6 @@
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="..\System.IO.Pipelines\System.IO.Pipelines.csproj" />
-  </ItemGroup>
```

Comment:
I am surprised that these changes are necessary. If they are, shouldn't they be applied to all projects? For example, Buffers.Experimental.csproj?